src/Runner/Runner.php

```diff
@@ -266,7 +267,7 @@ private function fixFile(\SplFileInfo $file, LintingResultInterface $lintingResu
             }
         }
 
-        $this->cacheManager->setFile($name, $new);
+        $this->cacheManager->setFileHash($name, $newHash);
```

Comment:
this is the optimization part, when we already had `newHash` calculated (line 158) and there is no need to re-calculate it